app/scripts/templates/welcome.html

```diff
@@ -0,0 +1,17 @@
+<h1>Welcome y'all</h1>
+
+{{#supportedBrowser}}
+<div id="install-container">
+  <p>Some details about the add-on</p>
+  <button class="install">Install the add-on</button>
+</div>
+<div id="continue-container" style="display: none;">
```

Comment:
Yeah I'm always torn about the right thing to do here. Using the existing utility classes is probably a good idea.